docs/GOVERNANCE.md

```diff
@@ -0,0 +1,207 @@
+# Repository Governance
```

Comment:
The doc makes no mention of OWNERS files, which is the primary mechanism by which we will enforce governance, as well as the procedures to approve/review the componetns.

Mention who get to be approvers, reviewers - whom are the root maintainers (root owners file) etc. 

This section should be something we could link to people to give them a quick over view of how the file system works, with `/approve` and `/lgtm` tide mechanics.